src/renderer/components/Reactions/styles.module.scss

```
@@ -0,0 +1,44 @@
+.reactions {
+  cursor: pointer;
+  margin-right: 10px;
+  position: relative;
+  top: 20px;
+}
+
+.emoji {
+  background-color: var(--messageIncomingBg);
+  border-radius: 24px;
+  border: solid 1px rgba(100, 100, 100, 0.5);
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+  font-size: 16px;
+  padding: 2px 6px;
+
+  & ~ & {
+    margin-left: 2px;
+  }
+
+  &.isFromSelf {
+    background-color: var(--messageOutgoingBg);
+  }
+
+  &.showMore {
+    color: var(--messageText);
+
+    &::after {
+      content: '⋯';
+      display: inline-block;
+      position: relative;
+      top: 1px;
+    }
+  }
+}
+
+.emojiCount {
+  bottom: 1px;
+  color: var(--messageText);
+  font-size: 11px;
+  font-weight: bold;
```

bold looks, well, a bit bold :) instead, use a slightly larger font size. see above for comparison

```suggestion
  font-size: 12px;
```